openrouteservice/src/main/java/heigit/ors/routing/RouteResultBuilder.java

```diff
@@ -110,6 +110,10 @@ public RouteResult createRouteResult(List<GHResponse> routes, RoutingRequest req
 			}
 		}
 
+		if (request.getSkipSegments() != null && !request.getSkipSegments().isEmpty()) {
+			result.addWarning(new RouteWarning(3));
```

Comment:
Use the RouteWarning.SKIPPED_SEGMENTS constant here, don't pass in a number itself